### PR TITLE
Configurable FR/Flow edit ranges

### PR DESCRIPTION
**Pull request by vovodroid**
Currently speed and flow adjustment menus have very wide range 10%-999%, thing that seems to be quite unreal and makes adjustments unhandy, especially by slider in touch screen.

This PR suggests following ranges:
**speed**: 30%-300%
**flow**: 50%-150%

Any other reasonable values are welcome.

### CODE REVIEW

**Comment by thisiskeithb** [comment]
I agree that the current limits are not ideal on touch screens, but I would suggest these limits be turned into config options with current values. Users will then be able to reduce them as they choose instead of us picking new, arbitrary values.

**Comment by vovodroid** [comment]
> I would suggest these limits be turned into config options with current values.

Do you mean Config or Config_adv.h? Or via menu?

**Comment by thisiskeithb** [comment]
> Do you mean Config or Config_adv.h?

Compile-time options in `Configuration.h` or `Configuration_adv.h`. Both files are huge now, so I don't really think it matters which one as long as it makes sense which section they end up in.

**Comment by vovodroid** [comment]
Ok, I'll add. What is procedure to add new defines? PR here or to configuration repository (folder "default")?

**Comment by vovodroid** [comment]
Well, I updated code with defines keeping original 10..999 range. Any way I would limit ranges for something practical, e.g. 50..200, or 25..400 (though I don't think later range has real usage).

File: Marlin/Configuration_adv.h [hunk 1]
```diff
@@ -1625,6 +1625,15 @@
 
 #endif // HAS_DISPLAY
 
+#if ANY(HAS_MARLINUI_MENU, DWIN_CREALITY_LCD, DWIN_LCD_PROUI, MALYAN_LCD, TOUCH_SCREEN)
+  #define SPEED_EDIT_MIN    10  // (%) Feedrate percentage edit range minimum
+  #define SPEED_EDIT_MAX   999  // (%) Feedrate percentage edit range minimum
```
Comment (thisiskeithb) [inline Marlin/Configuration_adv.h]: Maximum, not minimum

File: Marlin/Configuration_adv.h [hunk 2]
```diff
@@ -1625,6 +1625,15 @@
...
+  #if HAS_EXTRUDERS
+    #define FLOW_EDIT_MIN   10  // (%) Flow percentage edit range minimum
+    #define FLOW_EDIT_MAX  999  // (%) Flow percentage edit range minimum
```
Comment (thisiskeithb) [inline Marlin/Configuration_adv.h]: Maximum, not minimum

**Pull request merged**